ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -587,6 +587,22 @@ DFSS:
   - discharge_fan_run_command
   - discharge_fan_run_status
 
+DF2SS:
+  description: "Two discharge fan run command and status (start/stop). With dampers"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - supply_air_damper_status_1
+  - return_air_damper_status_1
+  - supply_air_damper_status_2
+  - return_air_damper_status_2
+  uses:
+  - discharge_fan_run_command_1
```

Comment:
These should be put on their own in a type called DFSS2X, to conform with the existing convention.

Reply:
Modified